packages/cli-kit/assets/cli-ruby/lib/project_types/theme/commands/common/root_helper.rb

```diff
@@ -49,12 +49,14 @@ def exist_and_not_empty?(root)
         private
 
         def current_directory_confirmed?
-          raise "Current theme directory can't be confirmed during tests" if @ctx.testing?
+          message = @ctx.message("theme.current_directory_is_not_theme_directory")
+          @ctx.abort(message) unless ShopifyCLI::Environment.interactive?
 
+          message += " " + @ctx.message("theme.confirm_current_directory")
```

Comment:
/supernit, this is more idiomatic plus we're explicitly showing we're adding a new line 

```suggestion
          message += " \n#{@ctx.message('theme.confirm_current_directory')}"
```